cmd/courier/main.go

```diff
@@ -87,51 +89,61 @@ func main() {
 	}
 
 	// configure our logger
-	logrus.SetOutput(os.Stdout)
-	level, err := logrus.ParseLevel(config.LogLevel)
+	loggerLevel := new(slog.LevelVar)
```

Comment:
do we need this? or can we just parse the `Level` value and then create the handler?

Reply:
using the levelVar is the only approach to support later changing the log level, but we change to use the parsed level and create a new handler.